<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwin</class>
 <widget class="QWidget" name="mainwin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>539</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
        background-color: #2E2E2E;
        font-family: Arial, sans-serif;
        }

        QPushButton {
        border: 2px solid #444;
        border-radius: 5px;
        background-color: #333;
        color: #FFFFFF;
        padding: 8px 15px;
        margin: 5px;
        }

        QPushButton:hover {
        background-color: #FF5733;
        }

        QPushButton:pressed {
        background-color: #C70039;
        }

        QLineEdit {
        border: 2px solid #444;
        border-radius: 5px;
        padding: 8px;
        background-color: #1C1C1C;
        color: #FFFFFF;
        margin-right: 10px;
        }

        QLineEdit:focus {
        border-color: #FF5733;
        background-color: #333;
        }

        QCalendarWidget QToolButton {
        height: 30px;
        width: 70px;
        color: #FFFFFF;
        font-size: 14px;
        icon-size: 20px, 20px;
        background-color: #FF5733;
        border: none;
        margin: 5px;
        }

        QCalendarWidget QToolButton:hover {
        background-color: #C70039;
        }

        QCalendarWidget QMenu {
        width: 100px;
        background-color: #333;
        color: #FFFFFF;
        }

        QCalendarWidget QSpinBox {
        width: 50px;
        font-size: 14px;
        color: #FFFFFF;
        background-color: #1C1C1C;
        selection-background-color: #FF5733;
        selection-color: #000000;
        }

        QCalendarWidget QWidget#qt_calendar_navigationbar {
        background-color: #333;
        }

        QCalendarWidget QTableView {
        border-width: 1px;
        border-color: #444;
        border-style: solid;
        border-radius: 5px;
        gridline-color: #444;
        background-color: #1C1C1C;
        }
            QCalendarWidget QHeaderView::section {
            background-color: #333;
            color: #FFFFFF;
            padding: 5px;
            border: 1px solid #444;
            }</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="searchLineEdit"/>
    </item>
    <item>
     <widget class="QListWidget" name="repetitorListWidget"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>Тесты</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Аккаунт</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>331</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
